### ISSUE TITLE
Warnings on IOS 8.4 when using library

### ISSUE THREAD

**Issue by freddiescott** [Telerik-Verified-Plugins/BarcodeScanner#10]
We have a project that has been using this library for quite some time.
Recently we noticed that we are getting warnings like these :

```
WARNING: -[<AVCaptureVideoPreviewLayer: 0x74423ac0> isOrientationSupported] is deprecated.  Please use AVCaptureConnection's -isVideoOrientationSupported
WARNING: -[<AVCaptureVideoPreviewLayer: 0x74423ac0> setOrientation:] is deprecated.  Please use AVCaptureConnection's -setVideoOrientation:
WARNING: -[<AVCaptureVideoPreviewLayer: 0x74423ac0> setOrientation:] is deprecated.  Please use AVCaptureConnection's -setVideoOrientation:
WARNING: -[<AVCaptureVideoPreviewLayer: 0x7023e940> isOrientationSupported] is deprecated.  Please use AVCaptureConnection's -isVideoOrientationSupported
WARNING: -[<AVCaptureVideoPreviewLayer: 0x7023e940> setOrientation:] is deprecated.  Please use AVCaptureConnection's -setVideoOrientation:
WARNING: -[<AVCaptureVideoPreviewLayer: 0x7023e940> setOrientation:] is deprecated.  Please use AVCaptureConnection's -setVideoOrientation:
```

Is is safe to say that Apple will at some stage (probably with iOS9) stop support for these properties and methods ?

**Comment by EddyVerbruggen** [comment]
Hi, yeah they will remove it at some stage but no one knows when. iOS 9 is unlikely because deprecations like these should have a long grace period to not suddenly break existing apps.

We'll make sure these get fixed, thanks!

**Comment by freddiescott** [comment]
Thanks for the reply Eddy !

**Comment by EddyVerbruggen** [comment]
These deprecation warnings have been fixed in 2.1.1.
